format_checker/forked-projects.json

```diff
@@ -356,5 +356,6 @@
   "https://github.com/yandex/yoctodb": "unforked",
   "https://github.com/yangfuhai/jboot": "unforked",
   "https://github.com/zalando/riptide": "unforked",
-  "https://github.com/zhangxd1989/spring-boot-cloud": "unforked"
+  "https://github.com/zhangxd1989/spring-boot-cloud": "unforked",
+  "https://github.com/opengoofy/hippo4j": "unforked"
```

Comment:
Please (1) keep projects sorted by URL (which also reduces the change by not adding `,` every time), and (2) maybe change the checker to check that `forked-projects.json` is sorted.